### DISCUSSION TITLE
When are bugfixes released?

### DISCUSSION THREAD

**Discussion by phil-lange** [Budibase/budibase#4857] [Help] [answered]
Hello,
What are the rules when you run the Selfhost release pipeline?
https://github.com/Budibase/budibase/actions/workflows/release-selfhost.yml

I'm waiting for this fix to be released: https://github.com/Budibase/budibase/pull/4710. I'm hosting using the budibase cli (docker-compose).

**Comment by shogunpurple** [comment, marked as answer]
Hi @phil-lange - releases are generally done in the first week of a month for the previous month. Just a few days delay on this one, but the February Release has just gone out for self hosters, and we will send out all the release notes over the coming days.

Thanks!
